levant/deploy.go

```diff
@@ -47,6 +47,12 @@ func (c *nomadClient) Deploy(job *nomad.Job, autoPromote int) (success bool) {
 		return
 	}
 
+	// If job.Type isn't set we can't continue
+	if job.Type == nil {
+		logging.Error("levant/deploy: Nomad job `type` is not set. Should be set to `service`")
```

Comment:
pedantic I know; but could the log messaged be change slightly? In particular `not set. Should` to `not set, should`

Reply:
No problem, change is pushed!